src/dataflow/src/render/reduce.rs

```diff
@@ -31,23 +76,427 @@ use super::context::Context;
 use crate::render::context::Arrangement;
 use crate::render::datum_vec::DatumVec;
 
-// This enum indicates what class of reduction each aggregate function is.
-#[derive(Clone, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
+/// This enum represents the three potential classes of aggregations.
```

Comment:
Maybe minor, and I don't know the reason, but why do we have the numbers there? Is that to make sure someone doesn't re-order them alphabetically and change some sorting somewhere, or for some other reason? If you happen to know, even if it wasn't yours, I'd jot it down as part of informing the reader what they are allowed to do here.

Reply:
that used to be important before we cleaned up basic aggregates to be fused together before collation, is not important anymore